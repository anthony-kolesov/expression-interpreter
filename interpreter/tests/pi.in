var n = 100000
var sequence = map({0, n}, i -> (-1.0)^i / (2 * i + 1))
var pi = 4 * reduce(sequence, 0, x y -> x + y)
print "n = "
out n
print ", pi = "
out pi
print "\n"
