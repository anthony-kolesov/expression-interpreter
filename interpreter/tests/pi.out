n = 100000, pi = 3.141603
